includes/class-freemius.php

```diff
@@ -6387,6 +6479,23 @@ function opt_in(
 		) {
 			$this->_logger->entrance();
 
+			if ( ! $is_uninstall && fs_request_is_action( $this->_slug . '_opt_in' ) ) {
```

Comment:
@fajardoleo this logic should definitely be in a separate method, probably name it `reconnect()`. And for consistency, I would name the `opt_in()` to `disconnect()`.